### ISSUE TITLE
Is the repo dead?

### ISSUE THREAD

**Issue by AustralEpitech** [ferdium/ferdium-server#70]
There is no commit since march. Is the repo dead?

**Comment by victorbnl** [comment]
No, it isn’t. The server is being rebuilt in https://github.com/ferdium/ferdium-server/pull/47, which has been updated last week. I believe this is such a huge pull request that it wouldn’t make sense to add features to the main repository, as these will get replaced anyway.

**Comment by SpecialAro** [comment]
Hi!

As @victorbnl said we are working hard on updating the Adonis dependency (which needs a full re-write of code from js to ts, and some other things).

Certainly in the future we will be able to work on new features and issues, rest assured 😁

I'll personally try to work on that too..

**Comment by AustralEpitech** [comment]
Oh that's awesome!
Should I keep the issue open, until the merge is done, for anyone else wondering?
